SmartCarMQTTApp/app/src/test/java/com/example/smartcarmqttapp/QuizUnitTest.java

```diff
@@ -0,0 +1,359 @@
+package com.example.smartcarmqttapp;
+
+import com.example.smartcarmqttapp.database.CrushersDataBase;
+import com.example.smartcarmqttapp.model.Question;
+import com.example.smartcarmqttapp.model.UserAnswer;
+import com.example.smartcarmqttapp.screens.quiz.QuizQuestionActivity;
+import com.example.smartcarmqttapp.state.QuizState;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.junit.Assert.*;
+import org.robolectric.Robolectric;
+import org.robolectric.RobolectricTestRunner;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@RunWith(RobolectricTestRunner.class)
+public class QuizUnitTest {
+
+    //fix this either using Robolectric testing (doesnt work atm) or something else
```

Comment:
Is this fixed or still todo?

Reply:
Fixed, as seen in line 23: I use Robolectric